diff -aur rebuilderd-0.22.0.old/tools/Cargo.toml rebuilderd-0.22.0/tools/Cargo.toml
--- rebuilderd-0.22.0.old/tools/Cargo.toml	2025-03-02 09:49:12.356221366 +0100
+++ rebuilderd-0.22.0/tools/Cargo.toml	2025-03-02 09:50:07.318732070 +0100
@@ -16,7 +16,7 @@
 ]
 
 [dependencies]
-bzip2 = "0.5"
+bzip2 = "0.4"
 chrono = { version = "0.4.19", features=["serde"] }
 clap = { version = "4", features = ["derive"] }
 clap_complete = "4.5.38"
